### PR TITLE
Validation of the publishing intervals fixed

### PR DESCRIPTION
**Pull request by MaidaB**
- Intervals can be 0 now
- Don't show TimeSpan.MinValue

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Models/ListNode.cs [hunk 1]
```diff
@@ -51,6 +51,6 @@ public class ListNode {
             {
                 return false;
             }
-        }
+        }
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Models/ListNode.cs:54]: Remove new trailing space.

File: samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs [hunk 2]
```diff
@@ -14,15 +14,15 @@ public class ListNodeValidator : AbstractValidator<ListNodeRequested> {
         public ListNodeValidator() {
             RuleFor(p => p.RequestedPublishingInterval)
                 .Must(BeAValidIntervalMs)
-                .WithMessage("Publishing interval cannot be less than 100 ms.");
+                .WithMessage("Publishing interval cannot be less than 0 ms.");
 
             RuleFor(p => p.RequestedSamplingInterval)
                 .Must(BeAValidIntervalMs)
-                .WithMessage("Publishing interval cannot be less than 100 ms.");
+                .WithMessage("Publishing interval cannot be less than 0 ms.");
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs:21]: Sampling interval, not publishing interval

File: samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs [hunk 3]
```diff
@@ -14,15 +14,15 @@ public class ListNodeValidator : AbstractValidator<ListNodeRequested> {
...
 
             RuleFor(p => p.RequestedHeartbeatInterval)
                 .Must(BeAValidIntervalSec)
-                .WithMessage("Heartbeat interval cannot be less than 1 second.");
+                .WithMessage("Heartbeat interval cannot be less than 0 second.");
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs:25]: how do I disable heartbeat?

File: samples/src/Microsoft.Azure.IIoT.App/src/Models/ListNodeRequested.cs [hunk 4]
```diff
@@ -4,40 +4,52 @@
 // ------------------------------------------------------------
 
 using Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models;
+using System;
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Models/ListNodeRequested.cs:7]: Move under namespace declaration as in rest of code.

File: samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs (hunk 3 above)
Comment (MaidaB) [reply samples/src/Microsoft.Azure.IIoT.App/src/Validation/ListNodeValidator.cs:25]: Just leave it empty

**Pull request merged**